PowerEditor/installer/nativeLang/swedish.xml

```diff
@@ -305,7 +306,8 @@
 					<Item id="50006" name="Sökvägskomplettering"/>
 				</Commands>
 			</Main>
-			<Splitter />
+            <Splitter>
+            </Splitter>
```

Comment:
@donho
I appreciate that you understand me and am willing to make exceptions to help me. I understand I've annoyed you, and it's absolutely anything I want. It's good that you keep your principles and have high ambitions.

Unfortunately, I've been on vacation, and I saw you already merged my PR.
Is it ok if I create a new PR that only sorts the strings and makes the file similar to the English source file? Then I promise not to annoy you anymore, and stick to the rules.